src/libraries/Microsoft.Extensions.Caching.Memory/src/Microsoft.Extensions.Caching.Memory.csproj

```diff
@@ -3,6 +3,12 @@
   <PropertyGroup>
     <TargetFrameworks>netstandard2.0;net461</TargetFrameworks>
     <EnableDefaultItems>true</EnableDefaultItems>
+<!--
+  Type 'Microsoft.Extensions.Caching.Memory.MemoryCache' exists on the previous stable version but not on the recent version.
+  Type 'Microsoft.Extensions.Caching.Memory.MemoryCacheOptions' exists on the previous stable version but not on the recent version.
+  Type 'Microsoft.Extensions.Caching.Memory.MemoryDistributedCacheOptions' exists on the previous stable version but not on the recent version.
+-->
```

Comment:
These look bogus.  We haven't removed these and if we did that'd not be something we'd be OK with baselining.  All of the missing types are the same.  We need to find out why those are being flagged.

Reply:
Is it true for all the deletions here ?

Reply:
Found the issue, the assembyMapper.GetNameSpaces is not returning the correct values, its missing the the namespace. Just debugging through the api compat to find the reason

Reply:
Its getting confused between fxResources.Microsoft.Extensions* and Microsoft.Extensions.* namespaces

Reply:
fix https://github.com/dotnet/sdk/pull/17799